tests/test_lap_archive_async.py

```
@@ -172,29 +170,170 @@ def test_archive_write_job_streams_trace_over_multiple_steps(tmp_path: pathlib.P
+          lap_ms = 82550,
+          is_pb = true,
+          is_valid = true,
+          trace = {},  -- no samples
+        }
+        local lapArchive = require("lap_archive")
+        _job, _err = lapArchive.createWriteJob(_opts, 50)
+        """
+    )
+    assert rt.globals()["_job"] is None, "createWriteJob must refuse an empty-trace lap"
+    err = rt.globals()["_err"]
+    assert isinstance(err, str) and "empty trace" in err, f"unexpected err: {err!r}"
```

Ruff (0.15.18)

[warning] Assertion should be broken down into multiple parts

(PT018)